src/UserService.Broker/Helpers/Login/CredentialsParser.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Net.Mail;
+using System.Text;
+
+namespace LT.DigitalOffice.UserService.Broker.Helpers.Login;
+public static class CredentialsParser
+{
+  public static bool IsEmail(this string value)
+  {
+    try
+    {
+      MailAddress m = new MailAddress(value);
```

Comment:
Why did you decide to use this name?

Reply:
i do not use the variable in other plase